project/project.go

```diff
@@ -15,6 +15,9 @@ import (
 	"github.com/docker/libcompose/yaml"
 )
 
+// ComposeVersion is name of docker-compose.yml file syntax supported version
+const ComposeVersion = "1.5.0"
```

Comment:
We might want to *up* that sometimes as we support compose file v2 which is... way older than `1.5.0`👼